NEWS.md

```diff
@@ -12,6 +12,8 @@
 
 2. `fwrite()` respects `dec=','` for timestamp columns (`POSIXct` or `nanotime`) with sub-second accuracy, [#6446](https://github.com/Rdatatable/data.table/issues/6446). Thanks @kav2k for pointing out the inconsistency and @MichaelChirico for the PR.
 
+3. Subsetting or aggregating columns of type `expression` works, [#5596](https://github.com/Rdatatable/data.table/issues/5596). Thanks to @tsp for the report, and @ben-schwen for the fix.
```

Comment:
I guess we should clarify what operations are/aren't supported here. E.g. this still won't work:

```r
dt = data.table(a=1:2, b=expression(1,2))
dt[,b[1],a]
# Type 'expression' is not supported by GForce head/tail/first/last/`[`
```

Is it worth adding support for that, or just specifying when "expression-in-`j`-with-`by`" works?

Reply:
I would see this as different issue to not depatch to GForce in the first place for type `expression`

Reply:
I'm not sure anyone actually needs this anyway; we can revisit upon request.